### PR TITLE
WIP 1.0.0 bug fixes

### PR DESCRIPTION
**Pull request by Magicode1**
For Comparison. Do not merge.
...

### CODE REVIEW

**Comment by archie426** [comment]
Oh gawd... the conflicts

**Comment by Magicode1** [comment]
Yeah. This is gonna be a nightmare

**Comment by jasonwynn10** [comment]
The nightmare isn't over yet! we have a while until pmmp API3 is completed

**Comment by jasonwynn10** [comment]
Does this PR _really_ need to be open right now? We aren't going to have to merge the branch for a long time.

**Comment by archie426** [comment]
@jasonwynn10 We can always close it and reopen it

File: src/revivalpmmp/pureentities/entity/monster/walking/SnowGolem.php [hunk 1]
```diff
@@ -2,38 +2,56 @@
...
     }
 
     public function getName(){
         return "SnowGolem";
     }
 
     public function targetOption(Creature $creature, float $distance) : bool{
-        return !($creature instanceof Player) && $creature->isAlive() && $distance <= 60;
+        if ($creature instanceof Player) {
+            if ($creature != null and $creature->getInventory() != null) { // sometimes, we get null on getInventory?! F**k
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/entity/monster/walking/SnowGolem.php]: Isn't this redundant? The function already checks if the variable is a Player

**Comment by kniffo80** [comment]
@jasonwynn10 will check ASAP. Currently I am visiting the father of my girlfriend. Returning home Sunday. I started with taming wolves Thursday. I'll recheck the complete code when home again ;-)

**Comment by jasonwynn10** [comment]
I think that we can try to merge the versions now. pmmp's 1.0 branch was merged into the master

**Comment by archie426** [comment]
Okay, conflict time!

**Comment by kniffo80** [comment]
+1

File: src/revivalpmmp/pureentities/PureEntities.php [hunk 2]
```diff
@@ -92,6 +92,7 @@ class PureEntities extends PluginBase implements CommandExecutor {
     const BUTTON_TEXT_FEED  = "Feed";
     const BUTTON_TEXT_MILK  = "Milk";
     const BUTTON_TEXT_TAME  = "Tame";
+    const BUTTON_TEXT_SIT   = "Sit";
```
Comment (jasonwynn10) [inline src/revivalpmmp/pureentities/PureEntities.php]: I suppose that "Ride" is next?

**Comment by kniffo80** [comment]
Anyone volunteer to merge that stuff? If not, I can try to merge this f**king stuff :expressionless:

**Comment by kniffo80** [comment]
I'm currently merging 1.0.0-bug-fixes into master locally. I hope I can finish that today!

**Pull request closed without merging**